.github/workflows/integration-tests-workflow.yml

```diff
@@ -109,6 +109,15 @@ jobs:
           node-version: ${{ steps.versions_step.outputs.node_version }}
           registry-url: 'https://registry.npmjs.org'
 
+      - name: Configure OpenSearch Dashboards
+        run: |
+          rm -rf ./config/opensearch_dashboards.yml
+          cat << 'EOT' > ./config/opensearch_dashboards.yml
+          server.host: "0.0.0.0"
```

Comment:
I added this dashboards configuration because the current cypress framework is having trouble determine among `localhost`, `127.0.0.1`, and `::`.

This is a known issue for cypress + node version 18+: https://github.com/cypress-io/cypress/issues/25397#issuecomment-1384096677